components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -57,15 +56,29 @@ export default function Preferences() {
                 const updatedUser = await getGitpodService().server.getLoggedInUser();
                 setUser(updatedUser);
 
-                toast("Your default workspace timeout was updated.");
+                let toastMessage = <>"Your default workspace timeout was updated."</>;
+                if (billingMode.data?.mode === "usage-based") {
+                    if (!billingMode.data.paid) {
+                        toastMessage = (
+                            <>
+                                {toastMessage} Note that this will only affect workspaces in paid organizations. Go to{" "}
```

Comment:
**thought**: I still find the fact this applies only to workspaces within orgs very confusing. We should seriously consider either 🅰️  enabling this for all workspaces and let users control how they use the product and consume credits or :b: moving this setting to project settings and disabling editing it for projects within non-paid organizations. 🆘 Cc @loujaybee